extensions/yarn/README.md

```diff
@@ -42,6 +46,39 @@ In cases where Phylum still needs to process some of the packages, the command
 will exit with a warning **without** installing the packages. Once the analysis
 is complete, another attempt can be made.
 
-[phylum]: https://phylum.io
-[phylum-cli]: https://github.com/phylum-dev/cli
-[yarn]: https://yarnpkg.com/
+## Troubleshooting
+
+There are some common reasons this extension may fail.
+
+### Sandbox violations
+
+The underlying package manager may fail due to operating within the Phylum
+sandbox. These errors are usually displayed in the output and say something
+about not being able to access a particular file or network resource.
+
+One possibility is that this is a valid sandbox violation that has not been
+accounted for with an exception. If you believe this is the case, please
+[submit an issue][issue] or [contact us][contact] to discuss adding exceptions.
+
+Another possibility is that a malicious action was taken by one of the packages.
+Blocking such malicious actions is a primary goal for using this extension but
+it may be hard to know what is malicious and what is not. If you need help
+making that determination, please [contact us][contact] and provide the log
+output along with any other relevant details.
+
+[issue]: https://github.com/phylum-dev/cli/issues/new/choose
+[contact]: https://docs.phylum.io/support/contact_us
+
+### Package violations
+
+Packages analyzed by this extension may not pass the
+[Phylum default policy][policy]. When this happens the extension will prevent
+the requested action and instead print a list of policy violations.
+
+There is not currently a method for allowing policy violations discovered by
+this extension. Instead, inspect the findings and determine if the offending
+entries are truly necessary. If not, update your project dependencies and try
+again. If they are, then the only recourse is to try the action again without
+the Phylum package manager extension.
+
+[policy]: https://docs.phylum.io/knowledge_base/policy
```

Comment:
Why would this only be added to the yarn extension? Also this seems **extremely** verbose, I wouldn't expect more than one paragraph of this size for each of these headings. I doubt users care to this degree.

Reply:
As noted in the PR summary and annotated with a file-level reminder comment, the plan is to iron out the language once and update this PR with the same language added to each of the other package manager extensions.

Alternative wording is welcome. It has already been seen that documentation of this sort is desired by real users. The one that kicked off these changes observed that they spent considerable time attempting to get the `yarn` extension working and did not understand what to do with the failed output provided by it.